apps/web/src/common/components/select/SelectBox.vue

```diff
@@ -71,12 +71,12 @@ const handleClick = () => {
     height: 100%;
     padding: 0.5rem;
 
-    &.selected {
-        @apply bg-blue-200 text-gray-700;
-    }
-
     &:hover {
         @apply bg-blue-100 text-gray-700;
     }
+
+    &.selected {
+        @apply bg-blue-200 text-gray-700;
+    }
```

Comment:
suggestion: I think this bug is very easy to reoccur since developers usually don't  think the order of declaration is not affective. 
I recommend more explicit way.. for example:
```suggestion
    &:not(.selected):hover {
        @apply bg-blue-100 text-gray-700;
    }
    
    &.selected {
        @apply bg-blue-200 text-gray-700;
    }
```

Additionally, for mobile case, I also recommend to wrap this code in a '@media (hover: hover) {}' block. 
```suggestion
    @media (hover: hover) {
        &:not(.selected):hover {
           @apply bg-blue-100 text-gray-700;
        }
    }
    
    &.selected {
        @apply bg-blue-200 text-gray-700;
    }
```

Reply:
Thank you for your comment!!